packages/contracts/test/plugins/governance/admin/admin-setup.ts

```diff
@@ -29,14 +33,14 @@ describe('AdminSetup', function () {
   before(async () => {
     signers = await ethers.getSigners();
     ownerAddress = await signers[0].getAddress();
-    targetDao = await deployNewDAO(ownerAddress);
+    targetDao = await deployNewDAO(signers[0]);
```

Comment:
Same as above, why do we need to make names non-meaningful?

Reply:
it was bcoz ownerAddress is an address not a signer
and now these function and the factories require a signer hence signers[0]